infrastructure/terraform/components/reporting/scripts/sql/reports/completed_comms_report.sql

```diff
@@ -13,9 +14,7 @@ LEFT OUTER JOIN request_item_plan_status rip ON
     ri.clientid = rip.clientid
 WHERE
     ri.clientid = ? AND
-    ri.status IN ('FAILED', 'DELIVERED') AND
-    rip.status IN ('FAILED', 'DELIVERED', 'SKIPPED') AND
     (
-        DATE(ri.completedtime) = DATE(?) OR DATE(rip.completedtime) = DATE(?)
+        (rip.status IN ('FAILED', 'DELIVERED', 'SKIPPED') AND DATE(rip.completedtime) = DATE(?)) OR
+        (ri.status IN ('FAILED') AND rip.status is null AND DATE(ri.completedtime) = DATE(?))
```

Comment:
Minor point but I'd maybe replace `rip.status is null` with `rip. requestitemplanid is null` as an ID field should never be null (but the status may not always have the same limitation).